frontend-new/src/auth/services/Authentication.service.ts

```diff
@@ -196,7 +196,7 @@ abstract class AuthenticationService {
       console.debug("Token checked. Token is valid");
       return { isValid: true, decodedToken: decodedToken };
     } catch (error) {
-      console.error(new TokenError("Error decoding token", error as Error));
+      console.warn(new TokenError("Error decoding token", error as Error));
```

Comment:
I think this should be console.error.

Because we wan't to know why decoding failed, may be because someone tried to manually update the token.

What do you think?

Reply:
I like that.. updating